src/api/API.cpp

```diff
@@ -70,7 +70,7 @@ SimulationResults APIInternal::execute(
 
     Settings settings;
     auto& parameters = study_->parameters;
-    parameters.optOptions = optOptions;
+    parameters.optOptions << optOptions;
```

Comment:
```c++
parameters.optOptions = optOptions;
```
is replaced with : 
```c++
parameters.optOptions << optOptions;
```
I suppose that existing **operator<<** is meant to be used here.
@pet-mit : as you introduced this operator, what do you think ?

Reply:
ok for `<<`, it seems more coherent with CLI use

Reply:
Please see with @pet-mit, its his.

Reply:
following @guilpier-code 's proposal on [this](https://github.com/AntaresSimulatorTeam/Antares_Simulator/pull/2574) PR, this operator overrides all fields of the optimization options, except for the logs flag, on which it operates a logical OR (code [here](https://github.com/AntaresSimulatorTeam/Antares_Simulator/blob/f26648161d04f1bbac5b17008570bddea701d44a/src/libs/antares/optimization-options/options.cpp#L24))

Reply:
The operator << disappeared eventually.